### ISSUE TITLE
Cant scrub

### ISSUE THREAD

**Issue by Ibsn87** [NateMccomb/TeslaCamViewerII#17]
Hello,

I have modern powerful windows 10 Pc with a top of the line CPU and GPU and more then enough Ram to sink a ship.  I am trying to watch the videos either directly off the USB SSD Samsung that's normally stored in the model 3. And or I have tried to copy the videos onto the same SSD on my PC as the teslacam viewe.exer is located - no difference... 

Problem: The clips always open on the sentry trigger. But the image on all 4 cameras is frozen. So I re-select the minute that's already highlighted on the Select Event column and the video will play from the beginning of the sentry minute. 

Frustratingly, I cant scrub forward or backward or the images will pause but the timer keeps scrolling. This also happens if I slow down after speeding up the footage, and or reversing the footage. The only way to get the video to play, is to re-select one of the minutes on the RHS.

I have the latest VLC 64 bit player installed. (Just reinstalled it again now and I have the latest version of your software.) Have had the same issue since day one.

To see each trigger I have to load the folder, re-select the trigger minute video, and wait (1 minute) until the end of each and every event. Slow and tiring process. Then if I missed it, or want to review it a little more, I have to go back to the start of the minute and re-watch it all.

For reference, I have never donated to freeware until I saw this. I am a supporter, in need of support. :) 

Cheers 

Ian

**Comment by NateMccomb** [comment]
It sounds like TCV is still using Windows Media Player instead of VLC. Click on Application Settings (Gear Icon) and verify that VLC is selected under "Video Player". There should have been a prompt when you first open TCV that states you either don't have VLC installed correctly or that VLC is not selected as the default video player. 

WMP will only work correctly if you right click on a folder and Re-encode all files but that takes extra time and why I added support for VLC. This was a bug that Tesla introduced when they changed how they encode the videos back with their update 2019.24.

Also verify that when installing VLC that "Web Plugins" & "ActiveX Plugin" are both checked as optional components you wish to install.

If you are still having issues then send me a copy of the most recent TCViewerLog-2020-XX-XX.txt log file to TeslaCamViewer@Gmail.com

**Comment by Ibsn87** [comment]
Hi Nate,

This is fantastic support.

Thankyou for your help. I check my installs/will do as you suggested and get back to you when I have a spare moment over the coming week!

I hope others in the Tesla community have paid you for your software, you have done a great job, a must have tool for anyone running sentry/dash cam etc!

Talk soon and Thanks!!!!

Ian

On Mar 13, 2020, at 4:24 PM, Nate <notifications@github.com> wrote:

﻿

It sounds like TCV is still using Windows Media Player instead of VLC. Click on Application Settings (Gear Icon) and verify that VLC is selected under "Video Player". There should have been a prompt when you first open TCV that states you either don't have VLC installed correctly or that VLC is not selected as the default video player.

WMP will only work correctly if you right click on a folder and Re-encode all files but that takes extra time and why I added support for VLC. This was a bug that Tesla introduced when they changed how they encode the videos back with their update 2019.24.

Also verify that when installing VLC that "Web Plugins" & "ActiveX Plugin" are both checked as optional components you wish to install.

If you are still having issues then send me a copy of the most recent TCViewerLog-2020-XX-XX.txt log file to TeslaCamViewer@Gmail.com<mailto:TeslaCamViewer@Gmail.com>

**Comment by Ibsn87** [comment]
Hello Nate

Apologies for the very slow reply. Life got in the way. I finally took a look at my player, I had indeed installed 64bit VLC.

For whatever reason TCV had not picked up on it as a default player and was still windows media player. I did get the warning, but was unsure if it was a generic note, as I had already installed the latest version of VLC. (I did not go into settings to adjust the default player as I was unaware of what was happening in the background)

I have done a re-install of VLC and running your latest build, its automatically picked VLC as default and it works great.

Thanks for all that you do!

Regards

Ian

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
